src/diff-pluses-and-minuses/diff-pluses-and-minuses.css

```diff
@@ -1,9 +1,15 @@
 .addition > .source:before {
```

Comment:
made everything important since bitbucket sometimes applies the same selector `.source::before` with the same properties, like `content: '######################################'`

Reply:
The '####' is not coming from Bitbucket, it is coming from our `limit-line-length` feature that @avshalomt  implemented a while ago. I didn't catch that that approach would end-up breaking this other feature, and these `!important` rules will now break that one.

Ideally we have to come up with a way to implement both features without them conflicting with one another. I'll give it some thought and come back to this, if you have any ideas let me know.

For now I'll not merge this yet.

Reply:
ok thinking about adding a named class feature for both to `.source` classes, like other features do.
`class="source __rbb_diff_pluses_and_minuses`
`class="source __rbb_limit_line_length`

once in the code i could probably find another way to improve it.
i will definitively revert the changes in this css file